Rakefile

```diff
@@ -3,6 +3,9 @@ require "bundler/gem_tasks"
 require "rake/testtask"
 require "bump/tasks"
 
+# Pushing to rubygems is handled by a github workflow
+ENV['gem_push'] = 'false'
```

Comment:
and github does not use rake to push ?

Reply:
It uses `gem build` and `gem push` [here](https://github.com/zendesk/gw/blob/42bc77df860c3f0995506d4a7378fe6c4129675d/.github/workflows/ruby-gem-publication.yml#L38-L40).

Reply:
Sorry, this lacks some context. I've been discussing these changes with @bquorning.

Reply:
The `ENV` change is to prevent a (failed) gem push from happening (and potentially rollback etc) when the user runs `rake release` locally.

Eventually, we want to switch away from `rake release` completely, and automatically initiate the gem release when a changed version.rb is pushed to GitHub.